packages/fxa-auth-server/test/local/db.js

```diff
@@ -34,21 +34,12 @@ describe('db, session tokens expire:', () => {
     sessionTokenWithoutDevice: 2419200000,
   };
 
-  let results, pool, log, tokens, db;
+  let log, tokens, db;
 
   beforeEach(() => {
-    results = {};
```

Comment:
Most of this code removal was from a time when we had a database service vs direct queries using Knex.